### DISCUSSION TITLE
Synchronizing money across a BungeeCord network

### DISCUSSION THREAD

**Discussion by Leahcimkrob** [Zrips/CMI#7746] [Enhancement]
Please add the Money Sync as the developer of the MySQLPlayerDataBridge doesn't seem to be able to do this. We've been trying to solve it for 2 months now, but we keep getting billions of coins added or deleted.

https://github.com/brunyman/MPDB/issues/100#issuecomment-1474867644
https://www.spigotmc.org/threads/mysql-player-data-bridge-paid.71612/page-133#post-4562405

it keeps pointing to VaultManager

**Comment by mrfloris** [comment]
As per the text you've selected and deleted to post your own .. please don't use issue tickets for suggestions; but use the discussion section instead; https://github.com/Zrips/CMI/discussions
